### PR TITLE
Badge icons for EmailProofs + polished old icon + added one more random ERC721 icon

### PR DESCRIPTION
**Pull request by T-Damer**

### CODE REVIEW

File: src/components/badges/BadgeBlock.tsx [hunk 1]
```diff
@@ -53,7 +56,7 @@ function Badge({ proof }: { proof: BaseProof }) {
 
   return (
     <BadgeCard
-      top={<BadgeIcon />}
+      top={isEmailProof ? <EmailBadge /> : <BadgeIcon />}
```
Comment (backmeupplz) [inline src/components/badges/BadgeBlock.tsx:59]: Let's rename the `BadgeIcon` as well?

File: src/components/badges/BadgeBlock.tsx [hunk 2]
```diff
@@ -64,12 +67,12 @@ function Badge({ proof }: { proof: BaseProof }) {
...
             {proof instanceof ERC721Proof ? (
               <ContractName address={proof.contract} />
-            ) : proof instanceof EmailProof ? (
+            ) : isEmailProof ? (
               proof.domain
             ) : (
               'Unknown'
             )}{' '}
-            (derivative)
+            {isEmailProof ? '(email)' : '(derivative)'}
```
Comment (backmeupplz) [inline src/components/badges/BadgeBlock.tsx:75]: I think keeping `derivative` is better?

Comment (T-Damer) [reply src/components/badges/BadgeBlock.tsx:75]: Inside the design, we have `(email)`
image

File: src/components/badges/BadgeBlock.tsx [hunk 3]
```diff
@@ -64,12 +67,12 @@ function Badge({ proof }: { proof: BaseProof }) {
           <>
             {proof instanceof ERC721Proof ? (
```
Comment (smyzin) [inline src/components/badges/BadgeBlock.tsx:68]: Maybe it's better to set `proof instanceof ERC721Proof` in the `const` variable. As you did with `isEmailProof`

**Pull request merged**